<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPlaylistDialog</class>
 <widget class="QDialog" name="EditPlaylistDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Playlist</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="listWidget_list"/>
     <widget class="QListWidget" name="listWidget_songs">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox_show_temporary">
       <property name="text">
        <string>Show temporary items</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radioButton_replace">
         <property name="text">
          <string>Load it instead of the current playlist</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_append">
         <property name="text">
          <string>Append it to the current playlist</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_rename">
       <property name="text">
        <string>&amp;Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete">
       <property name="text">
        <string>&amp;Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>&amp;Load</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget_list</tabstop>
  <tabstop>radioButton_replace</tabstop>
  <tabstop>radioButton_append</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_rename</tabstop>
  <tabstop>pushButton_delete</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>EditPlaylistDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>EditPlaylistDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
